pyworkflow/protocol/protocol.py

```diff
@@ -706,10 +706,11 @@ def inputProtocolDict(self):
 
     def hasLinkedInputs(self):
```

Comment:
I'd also change this name or have 2 related methods:

hasLinkedInputs should return true or false if the is input linked

In this case the code inside is for something like a "isInputReady" function.